lib/pub_sub/railtie.rb

```diff
@@ -6,6 +6,7 @@ class Railtie < Rails::Railtie
 
     initializer 'pub_sub.initialize' do
       PubSub.config.logger = Rails.logger
+      PubSub.config.log_level = :debug
```

Comment:
so does this set the default log level unless overridden in the inititalizer file?